src/com/java/class01/FirstJavaClass.java

```diff
@@ -4,8 +4,6 @@ public class FirstJavaClass {
 	
 	public static void main(String[] args) {
 		System.out.println("Hello World");
-		System.out.println("Hello Java");
```

Comment:
Why did you remove this line? was it a requirement? Can you share the story here?

Reply:
User Story : CC-2336